cmd/health.go

```diff
@@ -68,10 +68,6 @@ Ready: Checks the readiness of an endpoint, which returns OK if the Prometheus s
 
 			p := perfdata.PerfdataList{
 				{Label: "statuscode", Value: 200},
-				{Label: "version", Value: info.Version},
-				{Label: "builddate", Value: info.BuildDate},
-				{Label: "builduser", Value: info.BuildUser},
```

Comment:
@RincewindsHat What doofus thought that was a good idea?

```
abbd95f5 (Markus Opolka     2022-12-15 14:07:02 +0100  70)                              {Label: "statuscode", Value: 200},
3712b402 (Markus Opolka     2022-11-18 09:18:04 +0100  71)                              {Label: "version", Value: info.Version},
3712b402 (Markus Opolka     2022-11-18 09:18:04 +0100  72)                              {Label: "builddate", Value: info.BuildDate},
3712b402 (Markus Opolka     2022-11-18 09:18:04 +0100  73)                              {Label: "builduser", Value: info.BuildUser},

```

Nevermind... see this is why `interface{}` was a bad idea from the beginning. Gotta change that

Reply:
🙃